doc/src/devdocs/pkgimg.md

```diff
@@ -0,0 +1,47 @@
+# Package Images
+
+Julia package images provide object (native code) caches for Julia packages.
+They are similar to Julia's [system image](@) and support many of the same features.
```

Comment:
```suggestion
They are similar to Julia's [system image](@dev-sysimg) and support many of the same features.
```

incomplete ref? Though there are other sections that might fit better than the [one I chose](https://docs.julialang.org/en/v1/devdocs/eval/#dev-sysimg).